scripts/tests/test_concept_mapping.sql

```
@@ -14,54 +14,54 @@ WITH observation_obs_src AS (
         'observation_source_concept_id' AS concept_field,
         COUNT(DISTINCT base.observation_source_concept_id) AS total_distinct,
         SUM(CASE WHEN base.observation_source_concept_id IS NOT NULL THEN 1 ELSE 0 END) AS total_rows,
-        COUNT(DISTINCT CASE WHEN cm.source_code_id IS NULL THEN base.observation_source_concept_id END) AS unmapped_concepts,
-        SUM(CASE WHEN base.observation_source_concept_id IS NOT NULL AND cm.source_code_id IS NULL THEN 1 ELSE 0 END) AS unmapped_rows
-    FROM "NCL_Data_Store_OLIDS_Alpha".OLIDS_COMMON.OBSERVATION base
-    LEFT JOIN "NCL_Data_Store_OLIDS_Alpha".OLIDS_TERMINOLOGY.CONCEPT_MAP cm ON base.observation_source_concept_id = cm.source_code_id
-    LEFT JOIN "NCL_Data_Store_OLIDS_Alpha".OLIDS_TERMINOLOGY.CONCEPT c ON cm.target_code_id = c.id
+        COUNT(DISTINCT CASE WHEN cm.source_concept_id IS NULL THEN base.observation_source_concept_id END) AS unmapped_concepts,
```

SQLFluff (4.2.1)

[error] Line is too long (128 > 80).

(LT05)